README.md

```diff
@@ -21,6 +21,19 @@ Developer Documents:
 * [Copyright and License Notices](./NOTICES.md)
 * [License](./LICENSE)
 
+## Configuration
+
+### RateLimit
+Basic rate limiting can be applied by supplying a `RateLimit` object in the `appsettings.json` file. If no `RateLimit` object is supplied, rate limiting is not configured for the application. Rate limiting (when applied) will be set globally and apply to all application endpoints. 
+
+The `RateLimit` object should have the following parameters.
+
+Parameter       | Description
+----------------|----------------
```

Comment:
I don't recall ever seeing a markdown table that doesn't have  leading and trailing pipes. But it works nicely here ✅.

The line wrapping comment above of course does not apply to the table.

The Google Style Guide for markdown suggests that this should be a list instead of a table, because of the line width. In this particular case, the table seems easily readable as is, and no need to change.